### DISCUSSION TITLE
No tests match the provided pattern

### DISCUSSION THREAD

**Discussion by Jokkata** [Cyfrin/foundry-full-course-cu#354] [General]
Hello.. i have a problem with the test in lesson 7.. When i make the test  "forge test --mt testFundFailsWithoutEnoughETH" i get this message 
[⠆] Compiling...
No files changed, compilation skipped

No tests match the provided pattern:
        match-test: `testFundFailsWithoutEnoughETH`
        
Someone help?

**Comment by shikhar229169** [comment]
This means you have not defined any test with the name testFundFailsWithoutEnoughETH
